app/bundles/CoreBundle/Assets/css/app/less/components/buttons.less

```diff
@@ -6,6 +6,7 @@
 // Base styles
 // ---------------------
 .btn {
+  transition: var(--all);
```

Comment:
I'd avoid changes in the less components. I take it those are copied from some library like Bootstrap. Can we overwrite this in the custom.less file instead?

Reply:
From what I understand, we have:

- a Bootstrap dependency located on node_modules, which results in its default look 
- LESS files inside CoreBundle/…/app/less overwriting bootstrap defaults with Mautic custom styles while keeping it organized

So, in fact, there are already 2 buttons.less files

I’d recommend to keep changes on its related LESS file from the CoreBundle folder since it wouldn’t be updated by NPM (⬆️ stability)

Reply:
I don't have the historical context to argue. But it's quite a lot of customisations to all the components. I wonder why we have custom.less then?

Why do we have the same change in app.css and libraries.css? It should be only in 1, right?

Reply:
`custom.less` seems to be the place to put new custom classes (completely different of and not present in Bootstrap)

Changes to `app.css` and `libraries.css` are a consequence of running `grunt compile-less`, Mautic's default LESS/CSS engine